apps/expo-go/metro.config.js

```diff
@@ -14,25 +14,10 @@ config.watchFolders = [
   __dirname, // Allow Metro to resolve all files within this project
   path.join(monorepoRoot, 'packages'), // Allow Metro to resolve all workspace files of the monorepo
   path.join(monorepoRoot, 'node_modules'), // Allow Metro to resolve "shared" `node_modules` of the monorepo
-  path.join(monorepoRoot, 'react-native-lab'), // Allow Metro to resolve `react-native-lab/react-native` files
```

Comment:
double confirm if we don't need react-native-lab for expo-go here?

Reply:
oh i see. it should be later resolved by #35381

Reply:
Yes exactly, I just removed it in this PR as it's kind of not-working properly. Then the follow-up PR solves that problem correctly (at least for the bundle of Expo Go).